purchase_duplicate_check/models/purchase_order_line.py

```diff
@@ -0,0 +1,88 @@
+from odoo import _, fields, models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = "purchase.order.line"
+
+    pending_order_ids = fields.Many2many(
+        "purchase.order",
+        string="Pending Orders",
+        compute="_compute_pending_order_ids",
+    )
+
+    def _compute_pending_order_ids(self):
```

Comment:
I think you do need at least `@api.depends('product_id')` here

Reply:
@geomer198 any comment?

Reply:
This compute method have many dependencies, that is why I don't write `depends` decorator here.

Reply:
> @geomer198 any comment?

This compute method have many dependencies, that is why I don't write `depends` decorator here.